src/Razor/test/testapps/SimpleMvc22/TagHelpers/EnvironementTagHelper.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace SimpleMvc22.TagHelpers
```

Comment:
Leave a comment stating that this is used for testing purposes to verify multiple TagHelpers applying to a single element